test/util/measurements_test.py

```diff
@@ -109,12 +111,10 @@ def _SaveResult(test_record):
     test.Configure(http_port=None)
     if self.UPDATE_OUTPUT:
       test.AddOutputCallbacks(_PickleRecord)
```

Comment:
oh hey, i almost forgot, @wallacbe was awesome and wrote this into a handy base class in `io/output/__init__.py`, so you can actually do this and remove `_PickleRecord`:
```Python
# At the top (might as well also clean this up):
RECORD_FILENAME = os.path.join(
    os.path.dirname(os.path.abspath(__file__)), 'measurements_record.pickle')

# Replace _LocalFilename('measurements_record.pickle') with RECORD_FILENAME.

# Instead of test.AddOutputCallbacks(_PickleRecord)
  test.AddOutputCallbacks(output.OutputToFile(RECORD_FILENAME))
```